contracts/Agreement.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity 0.4.15;
+
+import './Standards/IEthereumForkArbiter.sol';
+
+
+contract Agreement {
+
+    IEthereumForkArbiter public ethereumForkArbiter;
+    string public agreementUri;
+
+    event AgreementAccepted(address indexed accepter);
+
+    modifier acceptAgreement(address accepter) {
```

Comment:
One conceptual issue: should we have a generic `acceptAgreement()` method that allows to explicitly sign this agreement? 
OK we should not - if it is necessary it should be implemented in derived contract. Ideally acceptance should be a part of a transaction that has some substance to it like `lock` or `transfer`
Just leaving this rant here so I do not forget it and not raise it back :>

Reply:
I was thinking the same thing, have a modifier `implicitAccept` and a function `explicitAccept`. Note that current `acceptAgreement` does not even require the accepter to sign or even be involved.
We could also maintain state of who accepted and who didn't. At the moment all of this doesn't seem worth the complexity.